app-policy/checker/match_test.go

```diff
@@ -351,10 +352,12 @@ func TestMatchRuleNamespaceSelectors(t *testing.T) {
 	}}
 
 	store := policystore.NewPolicyStore()
-	id := proto.NamespaceID{Name: "src"}
-	store.NamespaceByID[id] = &proto.NamespaceUpdate{Id: &id, Labels: map[string]string{"place": "src"}}
-	id = proto.NamespaceID{Name: "dst"}
-	store.NamespaceByID[id] = &proto.NamespaceUpdate{Id: &id, Labels: map[string]string{"place": "dst"}}
+	id := types.NamespaceID{Name: "src"}
+	pid := types.NamespaceIDToProto(id)
```

Comment:
`pid` is a bit ambiguous. How about change it to `protoID`? This applies to other files in this PR.

Reply:
changed all `pid` to `protoID`.